app/src/main/java/io/apicurio/registry/storage/impl/sql/SQLServerSqlStatements.java

```diff
@@ -69,7 +69,7 @@ public String isDatabaseInitialized() {
     public String upsertContent() {
         return String.join(" ",
                 "MERGE INTO content AS target",
-                "USING (VALUES (?, ?, ?, ?, ?, ?)) AS source (contentId, canonicalHash, contentHash, content, artifactreferences)",
+                "USING (VALUES (?, ?, ?, ?, ?)) AS source (contentId, canonicalHash, contentHash, content, artifactreferences)",
```

Comment:
How was this working???

Reply:
It wasn't, the sql server variant was not being verified (not it is).